Extractor/Config.cs

```diff
@@ -642,6 +642,19 @@ public class SubscriptionConfig
         /// Log bad subscription datapoints
         /// </summary>
         public bool LogBadValues { get; set; } = true;
+
+        /// <summary>
+        /// Server namespace nodes the should be subscribed to a rebrowse upon changes to their values.
+        /// </summary>
+        public ServerNamespacesToRebrowseConfig? ServerNamespacesToRebrowse { get; set; }
+    }
+    public class ServerNamespacesToRebrowseConfig
+    {
+        /// <summary>
+        /// Desired namespace nodes to subscribe to.
+        /// </summary>
+        /// <typeparam name="ProtoNodeId"></typeparam>
+        public IEnumerable<ProtoNodeId> NamespaceNodes { get; set; } = new List<ProtoNodeId>();
```

Comment:
This is kinda silly. Why do we need to specify the NodeId of each namespace?

First of all, does it not make sense to just subscribe to all of them? For users like the one we met who had hundreds of namespaces, this may be quite clunky.

Secondly, namespaces have unique names, it is cleaner to just use those, rather than needing to note down the actual NodeId of each.

Reply:
I understand that users may have a lot of namespaces but giving them the option to provide a list of namespaces to subscribe to will future-proof this feature.
Personally, I think it would be better for users to provide the node id each namespace that way it's clarity as to what the user will be subscribing to.

Reply:
To me it kind of feels like it does the opposite of future-proofing it, i.e. we make assumptions about the usage that have not yet been proven? Any reason why we don't just have a true/false now, then optionally add a filter in the future?